dbms/src/Flash/Mpp/MPPTaskId.h

```diff
@@ -23,23 +23,45 @@ struct MPPTaskId
 {
     MPPTaskId()
         : start_ts(0)
+        , server_id(0)
+        , query_ts(0)
         , task_id(unknown_task_id){};
 
-    MPPTaskId(UInt64 start_ts_, Int64 task_id_)
+    MPPTaskId(UInt64 start_ts_, Int64 task_id_, UInt64 server_id_, UInt64 query_ts_, UInt64 local_query_id_)
```

Comment:
Can we wrap a new struct `MPPQueryId`?
```
struct MPPQueryId
{
UInt64 server_id;
UInt64 query_ts;
UInt64 local_query_id;
}
```
Then we can just use MPPQueryId as the key of the MPPQueryMap instead of generating a string.

Reply:
And then we can avoid `decode` and `encode`.